Test/Test.cs

```diff
@@ -562,13 +562,14 @@ public void TestVerificationCheckRequest()
         {
             var verificationCheckRequest = new VerificationCheckRequest
             {
-                ApiKey = "35d603c1513f2567:",
+                //Please provide the valid api key in place of 'key'
+                ApiKey = "key",
                 Code = 655543,
                 UserId = "vineethk@exalture.com"
 
             };
-
-            verificationCheckRequest.ApiKey = "35d603c1513f2567";
+            //Please provide the valid api key in place of 'key'
+            verificationCheckRequest.ApiKey = "key";
 
             Assert.Equal(Convert.ToBase64String(Encoding.Default.GetBytes("35d603c1513f2567")),
```

Comment:
Here should be 'key' instead of hardcoded key. This verification checks that what we passed as a key above is same as expected.